### PR TITLE
Changed the service schema and made changes in the api

### PR DESCRIPTION
**Pull request by ritwizsinha**

### CODE REVIEW

File: api/routes/services.js [hunk 1]
```diff
@@ -35,28 +35,24 @@ router.get("/", (req, res, next) => {
...
     });
 
-    validator.check().then(function (matched) {
-        if (!matched) {
-            res.status(422).json({ msg: validator.errors });
-        }
-        else {
+
             const vendor = req.user.id;
+            console.log(req.user,vendor);
```
Comment (harsh-jindal) [inline api/routes/services.js]: Remove this

File: api/routes/services.js [hunk 2]
```diff
@@ -202,20 +196,16 @@ router.delete('/vendors/:vendorId', checkAuth, (req, res) => {
 // Update a service by Id  (access: vendor)
 router.patch('/:serviceId', checkAuth, (req, res) => {
     const id = req.params.serviceId;
-    const { name, detail, rating } = req.body;
+//
```
Comment (harsh-jindal) [inline api/routes/services.js]: Remove this

File: api/routes/services.js (hunk 1 above)
Comment (ritwizsinha) [reply api/routes/services.js]: Done

**Comment by ritwizsinha** [comment]
Made the changes @harsh-jindal  @deepanjan05 please review

**Review by deepanjan05 — APPROVED** [review]

**Pull request merged**